picard/script/functions.py

```diff
@@ -1203,7 +1203,7 @@ def func_datetime(parser, format=None):
         return datetime.datetime.now(tz=local_tz).strftime(format)
     except ValueError:
         stackitem = parser._function_stack.get()
-        raise ScriptRuntimeError(stackitem, "Unsupported format code")
+        raise ScriptRuntimeError(stackitem, "Unsupported format code") from None
```

Comment:
Wouldn't the source `ValueError` be interesting here, because it could hint into what error the format string had? Also I'm not sure what actually causes a `ValueError` here.